test/CMakeLists.txt

```diff
@@ -61,11 +61,16 @@ list( APPEND test_output
   testoutput/tropomi_no2.nc
 )
 
+list( APPEND data
```

Comment:
Remove this part, but put the test small BUFR file in the testinput/ list above here.
Also, once the test produces some output, we should have a reference netCDF IODA file in testoutput/ and use nccmp to compare the results like the other ioda-converters do.